cookbooks/omnibus-supermarket/.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+Metrics/LineLength:
```

Comment:
If we're going to add a rubocop.yml, should we add it at the top level of the repo instead of in the cookbook?